notebooks/STIS/drizpac_notebook/pre-requirements.sh

```diff
@@ -1,6 +1,4 @@
 # Install numpy
-pip install --force-reinstall -v 'numpy<2.0.0'
-# Install stis_cti
-pip install --no-cache-dir stis_cti
+pip install numpy
```

Comment:
For our records:

We believe the `stis_cti` source build requires numpy to already be installed prior to the `pip install` call.